modules/op-disabling-cluster-tasks-and-pipeline-templates.adoc

```diff
@@ -29,3 +29,7 @@ spec:
         value: 'true'
 ----
 
+[IMPORTANT]
+====
+In {pipelines-title} 1.10, cluster task functionality is deprecated and is planned to be removed in a future release.
```

Comment:
```suggestion
In {pipelines-title} 1.10, ClusterTask functionality is deprecated and is planned to be removed in a future release.
```

Reply:
@koustav @vdemeester Should we mention OSP 1.16 or OSP 1.17 here instead of future releases if we are sure to remove in 1.16 or 1.17 ??

Reply:
Well, it will depend on upstream (and as it looks right now, one of us doing the change), so we either decide 1.16 or 1.17 or we can stay vague about it 🙃

Reply:
I'm OK with `future release` until we are 99% sure :)

Reply:
Replaced the functionality description in all cases of this note - I do see what you mean here